app/models/work.rb

```diff
@@ -0,0 +1,21 @@
+class Work < ApplicationRecord
+  has_many :votes
+  has_many :users, through: :votes
+
+  validates :title, presence: true
+
+  def self.top
+    Work.all.max_by do |work| work.votes.length end
+  end
+
+  def self.list_by_votes(category)
+    list = Work.where(category: category)
+    sorted_list = list.sort_by { |work| -work.votes.length }
+    return sorted_list
```

Comment:
This is very specifically a "bonus" refactoring: I think this will be fine with just:

```ruby
return list.sort_by { |work| -work.votes.length }
```